graf2d/gpad/inc/TPad.h

```diff
@@ -487,6 +487,7 @@ inline Int_t TPad::XtoAbsPixel(Double_t x) const
 //______________________________________________________________________________
 inline Int_t TPad::XtoPixel(Double_t x) const
 {
+   if (TMath::IsNaN(x)) return 0;
```

Comment:
Do you have a reproducer for that one ? such check in a such low level function might be a performance killer.....

Reply:
Hi @cout ,

No, I don't have a standalone reproducer ( I see it in zevis) and the origin of nans seems to be elsewhere.


Also, I would suggest considering this branch as WIP and just a common place where I can put all the "kind of fixes" so you can easily see them.


Best regards,

Andrii

Reply:
Found the problematic code. It was not in ROT, but just passed some garbage to ROOT functions. Remeved the fix.